admin_manual/enterprise_user_management/user_auth_shibboleth.rst

```diff
@@ -5,72 +5,36 @@ Shibboleth Integration (Enterprise Subscription only)
-2. Enable the Shibboleth app.
-3. Map Shibboleth environment variables to ownCloud attributes.
+1. Enable and configure the apache Shibboleth module.
+2. Enable and configure the ownCloud Shibboleth app.
 
-Currently supported installations are based on the `native Apache integration`_ 
-. The individual configuration of the service provider is highly dependent on 
-the operating system, as well as on the integration with the Identity 
-Providers (IdP), and require case-by-case analysis and installation.
-
-The ownCloud Desktop Client can interact with an 
-ownCloud instance running inside a Shibboleth Service Provider by using built-in 
-browser components for authentication against the IdP.
-
-The regular ownCloud Android and iOS mobile apps do not work with Shibboleth.
-However, customers who create 
-:doc:`branded mobile apps with ownBrander 
-<../enterprise_clients/creating_branded_apps>`
-have the option to enable SAML authentication in ownBrander.
-
-Enterprise customers also have the option to request a regular ownCloud 
-mobile client built to use Shibboleth from their ownCloud account 
-representatives.
-
-The ownCloud desktop sync client and mobile apps store users' logins, so 
-your users only need to enter their logins the first time they set up their 
-accounts. 
-
-.. note:: The ownCloud clients may use only a single Shibboleth login per 
-   ownCloud server; multi-account is not supported with Shibboleth.
-
-These screenshots show what the user sees at account setup. Figure 1 
-shows a test Shibboleth login screen from 
-`Testshib.org <https://www.testshib.org/index.html>`_ on the ownCloud desktop 
-sync client.
-
-.. figure:: ../images/shib-gui1.png
-   :alt: First client login screen.
+The apache Shibboleth module
+----------------------------
 
-   *figure 1: First login screen*
-   
-Then after going through the setup wizard, the desktop sync client displays the 
-server and login information just like it does for any other ownCloud server 
-connections.
+Currently supported installations are based on the `native Apache integration`_.
+The individual configuration of the service provider is highly dependent on
+the operating system, as well as on the integration with the Identity
+Providers (IdP), and require case-by-case analysis and installation.
 
-.. figure:: ../images/shib-gui4.png
-   :alt: The ownCloud client shows which server you are connected to.
+A good starting point for the service provider installation can be found in
+`the official Shibboleth wiki`_.
 
-   *figure 2: ownCloud client displays server information*
-   
-To your users, it doesn't look or behave differently on the desktop sync 
-client, Android app, or iOS app from an ordinary ownCloud account setup. The 
-only difference is the initial setup screen where they enter their account 
-login.
+A successful installation and configuration will populate apache environment
+variables with at least the unique user id which is then used by the ownCloud
```

Comment:
We should probably say 'a' unique user id. In shibboleth terms it is typically the eppn but I don't want to confuse users with yet another term for the same thing. Ultimately, the userid is used as the ownCloud internal `userid`. So we should probably also fix the UI and change `uid` to `userid`. It is the same as in the LDAP expert settings "Internal username attribute".